### ISSUE TITLE
Linux Shutdown Cron Job fails with error 127

### ISSUE THREAD

**Issue by beastarman74** [seriousm4x/UpSnap#37]
When setting a cron job using the sshpass command method to shutdown a Linux machine the job fails with error 127. As per below screenshot of the logs.

![Screenshot_20230201_002639](https://user-images.githubusercontent.com/44511355/215915067-2bc85707-753d-4cf4-bff6-fda281d6aad4.jpg)

command syntax used: sshpass -p <pw> ssh -o "StrictHostKeyChecking=no" <un>@192.168.1.17 "sudo poweroff"

Googling the error suggests that Error 127 is command not found.

I checked the syntax of the command I was using by running on the docker host that runs UpSnap. This worked successfully.

To try and diagnose the cause I used docker exec to go into the container and manually run the command. When running the command within the UpSnap container it throws a command not found error.

sshpass exists in the container but it looks like the 
ssh program, the other app used in the command is not included. Obviously this might be incorrect diagnosis as I am unfamiliar with the inner workings.

**Comment by beastarman74** [comment]
Sorry if I caused any confusion.

I am running the latest version. 3.0.2

I have run the command outside the container on the docker host of UpSnap. This works, and the target machine is set to shutdown.

I have then run the command inside the container and got the error command not found. 

sshpass definitely is in the container, its also on all the machines I am trying to shutdown too.

sudo is on all the target machines too.

i thought it might be ssh that is missing as when I run ssh inside the container it comes back not found.

**Comment by seriousm4x** [comment]
Hm thats weird. I use shutdown with sshpass as well and it works for me.
Is your host windows or linux?

**Comment by seriousm4x** [comment]
I just tried it on my Pi 4 and I think I've found the error.
```
/app # sshpass -p password ssh -o "StrictHostKeyChecking=no" max@ip "sudo poweroff"
SSHPASS: Failed to run command: No such file or directory
```
But I'm not seeing the error in the docker logs when I click poweroff in the webinterface.
Can you try to install openssh inside the container? That fixed it for me. `apk update && apk add openssh`

**Comment by beastarman74** [comment]
Yes that's done it. I thought it looked like it was ssh that wasn't missing.

**Comment by seriousm4x** [comment]
Thanks for reporting. I've released 3.0.3 :)
